tests/lib/rules/no-use-computed-property-like-method.js

```diff
@@ -0,0 +1,667 @@
+          methods: {
+            fn() {
+              this.computedReturnString
+              this.computedReturnNumber
+              this.computedReturnObject
+              this.computedReturnArray
+              this.computedReturnBoolean
+            }
+          }
+        }
+      </script>
+      `
+    },
+    {
+      filename: 'test.vue',
+      code: `
+      <script>
+        export default {
+          computed: {
+            computedReturnFunction() {
+              const fn = () => alert('computedReturnFunction')
+              return fn
+            }
+          },
+          methods: {
+            fn() {
+              this.computedReturnFunction
+              this.computedReturnFunction()
+            }
+          }
+        }
+      </script>
+      `
+    },
+    {
+      filename: 'test.vue',
+      code: `
+      <script>
+        export default {
+          computed: {
+            computedReturnMethodsReturnString() {
+              return this.methodsReturnString
+            },
+            computedReturnMethodsReturnNumber() {
+              return this.methodsReturnNumber
+            },
+            computedReturnMethodsReturnObject() {
+              return this.methodsReturnObject
+            },
+            computedReturnMethodsReturnArray() {
+              return this.methodsReturnArray
+            },
+            computedReturnMethodsReturnBoolean() {
+              return this.methodsReturnBoolean
+            }
+          },
+          methods: {
+            methodsReturnString() {
+              return 'methodsReturnString'
+            },
+            methodsReturnNumber() {
+              return 'methodsReturnNumber'
+            },
+            methodsReturnObject() {
+              return {
+                inside: "inside"
+              }
+            },
+            methodsReturnArray() {
+              return [1,2,3,4,5]
+            },
+            methodsReturnBoolean() {
+              return true
+            },
+            fn() {
+              this.computedReturnMethodsReturnString
+              this.computedReturnMethodsReturnNumber
+              this.computedReturnMethodsReturnObject
+              this.computedReturnMethodsReturnArray
+              this.computedReturnMethodsReturnBoolean
+            }
+          }
+        }
+      </script>
+      `
+    },
+    {
+      filename: 'test.vue',
+      code: `
+      <script>
+        export default {
+          computed: {
+            computedReturnMethodsReturnFunction() {
+              return this.methodsReturnFunction
+            }
+          },
+          methods: {
+            methodsReturnFunction() {
+              const fn = () => alert('methodsReturnFunction')
+              return fn
+            },
+            fn() {
+              this.computedReturnMethodsReturnFunction
+              this.computedReturnMethodsReturnFunction()
+            }
+          }
+        }
+      </script>
+      `
+    }
+  ],
```

Comment:
I have found an error in some cases. Could you add the following test cases?

```vue
<script>
export default {
  computed: {
    bar() {
      return
    }
  }
}
</script>
```

```vue
<script>
export default {
  methods: {
    fn() {
      this.foo()
    }
  }
}
</script>
```

Reply:
Thanks!
I'll do that :)

Reply:
@ota-meshi 
I changed some codes to support the above cases.
https://github.com/vuejs/eslint-plugin-vue/pull/1234/commits/3b77c4e3efc0dc07d773989d932a9231baf4f253
https://github.com/vuejs/eslint-plugin-vue/pull/1234/commits/ce93fa61ea03746bd724def323417d3e56c493c8

However, I will not support this case:
```vue
<script>
export default {
  methods: {
    fn() {
      this.foo()
    }
  }
}
</script>
```
Because this rule only checkes whether computed properties are expected as property or not.
In this case, computed is not used.

Reply:
I don't want support. I've found that using a rule in that source code reports a crash or an incorrect error, so I just want to add a test case to test if it's okay.

Reply:
Thanks.
I just make sense of what you said :)
I add test case
https://github.com/vuejs/eslint-plugin-vue/pull/1234/commits/7cb18fefa658a95006b7357dd96da0aeaf74814f